e2e/cypress/tests/integration/archived_channel/archive_channel_add_reaction_spec.ts

```diff
@@ -35,7 +35,7 @@ describe('Archived channels', () => {
         // # Get the last post for reference of ID
         cy.getLastPostId().then((postId) => {
             // # Click the add reaction icon
-            cy.clickPostReactionIcon(postId);
+            cy.clickPostReactionIcon(postId, 'CENTER');
```

Comment:
Could you please correct the second parameter (`location`) as optional so this can remain as is?